### PR TITLE
GroupMeta should get merged when merging groups

### PR DESCRIPTION
**Pull request by mattrobenolt**
Fixes GH-3411

@getsentry/infrastructure @drcapulet

### CODE REVIEW

**Comment by drcapulet** [comment]
LGTM 👍

**Comment by tkaemming** [comment]
Looks good here.

Might be worth adding `GroupMeta` to the other group prior to merging, and asserting that only one `GroupMeta` exists at the end of the test, and has the value from the group that absorbed the other group. (This is the actual/expected behavior, but probably good to reinforce this in tests.)

**Comment by mattrobenolt** [comment]
@tkaemming you mean in the tests?

**Comment by tkaemming** [comment]
Oops — yeah.

**Comment by mattrobenolt** [comment]
Sorry for the delay @drcapulet!

File: tests/sentry/tasks/test_merge.py [hunk 1]
```diff
@@ -134,7 +148,9 @@ def test_merge_with_group_meta(self):
         GroupMeta.objects.populate_cache([group1, group2])
 
         assert not GroupMeta.objects.get_value(group1, 'github:tid')
+        assert not GroupMeta.objects.get_value(group1, 'other:tid')
         assert GroupMeta.objects.get_value(group2, 'github:tid') == '134'
+        assert GroupMeta.objects.get_value(group2, 'other:tid') == 'abc'
```
Comment (tkaemming) [inline tests/sentry/tasks/test_merge.py]: :ok_hand:

**Pull request merged**